sphinx/_ext/.gitignore

```diff
@@ -1 +0,0 @@
-edit_on_github.pyc
```

Comment:
Just a heads up that on a non-clean build, removal of this file can lead to `scc merge` failing since untracked files are present:

```
(base) /opt/own-bf/bio-formats-build/bio-formats-documentation $git status
HEAD detached from 8847d26f
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	sphinx/_ext/
```

Reply:
Maybe a case for adding `_ext` or `*.pyc` to the top-level `.gitignore`

Reply:
done in ddd3721e937c1c270729dec0b9d987cbf34b578d